reflex-dom-core/src/Reflex/Dom/Prerender.hs

```diff
@@ -1,4 +1,5 @@
 {-# LANGUAGE ConstraintKinds #-}
+{-# LANGUAGE CPP #-}
```

Comment:
CPP is supposed to be the first language extension I thought?

Reply:
Does it make a difference? I wouldn't have expected that.

Reply:
It does if you want to enable/disable certain extensions with CPP...

Reply:
@3noch Aaaaand that's another why -XCPP is horrible :P

Reply:
```haskell
#if 0
{-# LANGUAGE CPP #-}
#endif
```

Reply:
@ryantrinkle Somehow I'm not comforted by the fact that this works.... :joy:

Reply:
Oh whew...it doesn't.

Reply:
:)